### DISCUSSION TITLE
[BUG] New-PlannerPlan - Group not found

### DISCUSSION THREAD

**Discussion by chris883773** [pnp/powershell#2060] [Q&A]
Hello,

I want to create a Planner for a site collection with powershell. 

I am doing the following:
```
Connect-PnpOnline "tester (Test)"

New-PnPPlannerPlan -Group "tester (Test)" -Title $PlannerTitle
```

i get an error with
```
New-PnPPlannerPlan : Group not found
In Zeile:1 Zeichen:9
+         New-PnPPlannerPlan -Group "tester (Test)" -Title $Planne ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [New-PnPPlannerPlan], PSArgumentException
    + FullyQualifiedErrorId : Argument,PnP.PowerShell.Commands.Planner.NewPlannerPlan


```
I tested with a user and a global admin. the user is member and the global admin is owner of the group. 

### What is the version of the Cmdlet module you are running?
ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Manifest   1.10.0     PnP.PowerShell                      {Add-PnPAdaptiveScopeProperty, Add-PnPPropertyBagValue, Copy-PnPFolder, Export-PnPClientSidePage...}
Manifest   1.9.0      PnP.PowerShell                      {Add-PnPClientSidePageSection, Add-PnPClientSideText, Add-PnPClientSideWebPart, Copy-PnPFolder...}
Manifest   1.3.0      PnP.PowerShell                      {Add-PnPClientSidePage, Add-PnPClientSidePageSection, Add-PnPClientSideText, Add-PnPClientSideWebPart...}

### Which operating system/environment are you running PnP PowerShell on?
- [x ] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by veronicageek** [comment]
@chris883773 - I can create a new plan with no issue. Your `Connect-PnPOnline` doesn't seem right though. 

![image](https://user-images.githubusercontent.com/25181757/176656047-31678517-f379-4197-a308-5ba5b4a3ec89.png)

**Comment by chris883773** [comment]
Which permissions your you user have on site collection? 


How do you connect to the PnpOnline?

**Comment by veronicageek** [comment]
My user was a global admin for that test, and you can follow this to install & connect to PnP PowerShell:
https://pnp.github.io/powershell/articles/authentication.html

**Comment by chris883773** [comment]
OKay. 

My connection was:
`Connect-PnPOnline -Url https://xxx.sharepoint.com/sites/tester -Interactive`
It is wrong? I use MFA for global Admin.
But i have the same Error.
